app/Livewire/Roadmap/Form.php

```diff
@@ -41,6 +42,16 @@ public function save()
         $feat->status_id = FeatureStatus::Draft;
         $feat->save();
 
+        if (!auth()->user()->can('vote', $feat)) {
```

Comment:
Something doesn't seem right ;-)

Reply:
Also, the admin panel has no policies, so ->can() will always return false (we only use policies in the app)